pkg/cluster/mpi/configmap.go

```diff
@@ -33,6 +33,11 @@ func (c configMapComponent) Reconcile(ctx *core.Context) (ctrl.Result, error) {
 			hostFileName: buildHostFile(cr),
```

Comment:
Hmmm, is this still used? Or was the `ConfigMap` unused before.

Reply:
Not sure if it's actually used, but I though it's a good idea to keep an "authoritative" hostfile generated on the server in case it's needed, because it's an essential piece of data for MPI. There is no harm to have it in any case. (If I am not mistaken, in Nucleus we ended up creating the source file be deducing names of the nodes that will be in the cluster.)

Reply:
(1) One map per "area of concerns"?
(2) I think the way it works now is bad. This should be a secret.

Reply:
> (1) One map per "area of concerns"?

Seems reasonable to me (1 is mounted in a client (or launcher before), 1 in a worker).